entrypoint.sh

```
@@ -43,10 +43,26 @@ fi
 docker_run="$docker_run -d --name mysql -p $INPUT_HOST_PORT:$INPUT_CONTAINER_PORT mysql:$INPUT_MYSQL_VERSION --port=$INPUT_CONTAINER_PORT"
 docker_run="$docker_run --character-set-server=$INPUT_CHARACTER_SET_SERVER --collation-server=$INPUT_COLLATION_SERVER --default-authentication-plugin=$INPUT_AUTHENTICATION_PLUGIN"
 
-sh -c "$docker_run"
+if ! sh -c "$docker_run"; then
+  echo "Failed to start MySQL container"
+  exit 1
+fi
 
+MAX_TRIES=120
+TRIES=0
 while ! docker exec mysql mysql -h"127.0.0.1" -P"$INPUT_HOST_PORT" -u"$HEALTHCHECK_USER" -p"$HEALTHCHECK_PASS" -e "SELECT 1" $INPUT_MYSQL_DATABASE &> /dev/null; do
```

Shellcheck (0.11.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)